### ISSUE TITLE
App force closes

### ISSUE THREAD

**Issue by crizant** [OpenFlutter/flutter_image_compress#39]
Hi, I tried including `flutter_image_compress: ^0.3.1` to my `pubspec.yaml` without writing any code,
then `flutter run`, the app successfully builds but force closes on launch when running on real android devices.

I tried adding `ext.kotlin_version = '1.3.21'` to `build.gradle` like this:
```
buildscript {
    ext.kotlin_version = '1.3.21'
    repositories {
        google()
        jcenter()
    }

    dependencies {
        classpath 'com.android.tools.build:gradle:3.2.1'
        classpath 'com.google.gms:google-services:4.2.0'  // Google Services plugin
    }
}
```

But same result.
I'm new to flutter and app development, did i missed something?

**Comment by CaiJingLong** [comment]
I don't know what's wrong with you.
I created a brand new project and added plugins. Without any modifications, I could run it successfully.
Add code, everything is normal.

[use_compress_example.tar.gz](https://github.com/OpenFlutter/flutter_image_compress/files/3071700/use_compress_example.tar.gz)

1. Download and extract the code.
2. cd use_compress_example.
3. flutter packages get.
4. flutter run.

**Comment by CaiJingLong** [comment]
You can see your error log in logcat.

**Comment by crizant** [comment]
I found the step to reproduce it:
pubspec.yaml
```
flutter_image_compress: ^0.2.4
flutter_sound: ^1.3.7
```

This will make the app crash.

**Comment by CaiJingLong** [comment]
You crashed when you were flutter run?
Or use flutter build apk, then install APK and crash?

**Comment by CaiJingLong** [comment]
I added these two plugins without crash.
Whether compress is `0.3.1` or `0.2.4`.

[use_compress_example.tar.gz](https://github.com/OpenFlutter/flutter_image_compress/files/3071835/use_compress_example.tar.gz)

**Comment by crizant** [comment]
Sorry, one more step:
add the following lines in `android/gradle.properties`:
```
android.useAndroidX=true
android.enableJetifier=true
```

**Comment by crizant** [comment]
And yes I crashed when using `flutter run`, not tried with APK.

**Comment by CaiJingLong** [comment]
Please use `0.3.1`.
This version of the library does not depend on support or AndroidX.
The Android part of this plugin has no dependencies other than Android SDK and kotlin.

The AndroidX problem can be seen [here](https://caijinglong.gitbooks.io/migrate-flutter-to-androidx/content/), but this is not related to this library.

**Comment by crizant** [comment]
Here is the whole list of dependencies I am using, and I am still investigating which causes the problem:
```
  firebase_core: ^0.3.1+1
  firebase_auth: ^0.8.1+4
  firebase_database: ^2.0.2
  flutter_facebook_login: ^2.0.0
  firebase_storage: ^2.1.0+1
  intl: ^0.15.7
  flutter_linkify: ^2.0.0
  url_launcher: ^5.0.2
  font_awesome_flutter: ^8.4.0
  photo_view: ^0.2.2
  flutter_advanced_networkimage: ^0.4.15
  image_picker: ^0.5.0+9
  nanoid: ^0.1.0
  flutter_image_compress: ^0.2.4
  flutter_sound: ^1.3.7
```

**Comment by crizant** [comment]
It seems the issue comes from `flutter_facebook_login: ^2.0.0`, it happens with or without the AndroidX thing.

**Comment by crizant** [comment]
I don't know why, but `flutter_facebook_login` works fine alone, it only crashes after adding `flutter_image_compress 0.3.1`.

**Comment by crizant** [comment]
Closing this issue because it seems related to `flutter_facebook_login`.
I solved it by using `flutter_facebook_login: 1.2.0`.
